poly_commit/src/kzg/kzg_structs.rs

```diff
@@ -0,0 +1,131 @@
+use arith::FieldSerde;
+use halo2curves::{pairing::Engine, CurveAffine};
+
+use crate::StructuredReferenceString;
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub struct KZGCommitment<E: Engine>(pub E::G1Affine);
+
+impl<E: Engine> Default for KZGCommitment<E>
+where
+    E::G1Affine: CurveAffine<ScalarExt = E::Fr, CurveExt = E::G1>,
+{
+    fn default() -> Self {
+        Self(E::G1Affine::default())
+    }
+}
+
+/// Structured reference string for univariate KZG polynomial commitment scheme.
+/// The univariate polynomial here is of coefficient form.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct CoefFormUniKZGSRS<E: Engine> {
+    /// power of \tau times the generators of G1, yielding
+    /// \tau^i over G1 with i ranging in \[ 0, 2^n - 1 \]
+    pub powers_of_tau: Vec<E::G1Affine>,
+    /// \tau over G2
+    pub tau_g2: E::G2Affine,
+}
+
+impl<E: Engine> Default for CoefFormUniKZGSRS<E>
+where
+    E::G2Affine: CurveAffine<ScalarExt = E::Fr, CurveExt = E::G2>,
+{
+    fn default() -> Self {
+        Self {
+            powers_of_tau: Vec::default(),
+            tau_g2: E::G2Affine::default(),
+        }
+    }
+}
+
+impl<E: Engine> StructuredReferenceString for CoefFormUniKZGSRS<E>
+where
+    <E as Engine>::G1Affine: FieldSerde,
```

Comment:
will need to change to `ExpSerde` as #214 is merged

Reply:
good point, see https://github.com/PolyhedraZK/Expander/pull/191/commits/41e4c9f9927f3337fcd9a85cf6c13ec5309a0783